### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by Eternalgratis**
- Created a table named owners with columns
- Created a table named species with columns
- Modified animals table
- Inserted data into the owner's table
- Inserted data into the specie's table
- Modified inserted animals to include the species_id value
- Modified inserted animals to include owner information (owner_id)
- Written queries (using JOIN) to answer the questions
## Query screenshot taken
vet-clinic-answers-to-query
vet-clinic-answers-to-quey2

vet-clinic-answers-to-query3

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -13,12 +13,33 @@ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALU
 ('Blossom', '1998-10-13', 3, true, 17),
 ('Ditto', '2022-05-14', 4, true, 22);
 
+INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34),
+('Jennifer Orwell', 19),
+('Bob', 45),
+('Jennifer Orwell', 19),
```
Comment (danmainah) [inline data.sql]: - [ ] Kindy remove this entry, you have already added it as the first value.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Eternalgratis, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
- [ ] Kindly customize the title, description and the getting started section of your readme file.

- [ ] Kindy add screenshots for all your queries results, this is a project requirement.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: data.sql [hunk 2]
```diff
@@ -13,12 +13,22 @@ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALU
...
```
Comment (elmar8287) [inline data.sql]: - [ ] Please, consider fixing the issue marked by the previous reviewer, about removing the doubled entry `('Jennifer Orwell', 19),`

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights

- The README file is modified as suggested 👍 
 
## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 2 above)
Comment (Eternalgratis) [reply data.sql]: Done

**Review by cynthiainga — APPROVED** [review]
Hi @Eternalgratis, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] Descriptive PR ✔️ 
- [ ] Good commit messages ✔️ 
- [ ] Used correct Git flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**